tests/test_coordinator.py

```
@@ -118,11 +119,17 @@ async def test_check_device_unique_id_mismatch_triggers_issue(
     # patch issue registry and async_shutdown to avoid side effects
     called = {"issue": 0, "shutdown": 0, "issue_kwargs": None}
 
-    async def fake_shutdown():
+    async def fake_shutdown() -> None:
+        """Record that the coordinator requested shutdown after repeated mismatches."""
         called["shutdown"] += 1
 
-    def fake_async_create_issue(**kwargs):
+    def fake_async_create_issue(**kwargs: Any) -> None:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)